src/objects/conf/base.py

```diff
@@ -107,22 +100,24 @@
 
 # setup_configuration command
 # sites config
-SITES_CONFIG_ENABLE = config("SITES_CONFIG_ENABLE", default=True)
-OBJECTS_DOMAIN = config("OBJECTS_DOMAIN", "")
-OBJECTS_ORGANIZATION = config("OBJECTS_ORGANIZATION", "")
+SITES_CONFIG_ENABLE = config("SITES_CONFIG_ENABLE", default=True, add_to_docs=False)
```

Comment:
Hmm I feel like a missing opportunity that we won't update `django-setup-configuration` config documentation with `generate_envvar_docs` command. 

Should we run a command to update it also inside `bin/generate_envvar_docs.sh`  ? 
Oh maybe in the future we can generate a separate page for them with it? 
Do you think this makes sense or I overcomplicate it?

Reply:
I did not include the setup_configuration envvars yet, because there already was detailed documentation for them and they're not shared between projects. My goal here was mainly to remove the need to duplicate the same help texts for the shared variables.

I do think it would be good to include the setup_configuration envvars as well at some point, though I do think it's best to keep it as a separate page (like it is now) to differentiate between them.

I'm not yet sure what the easiest way to do that, I guess we could indeed run the command with a different `--file` and exclude the other groups. We'd probably also have to specify another template for it though, to add the intro text etc. Maybe it should be a separate issue?

Reply:
Yeah, and we can investigate in the separate issue that maybe Paul's doc generation is enough and covers all our needs?

Reply:
created https://github.com/maykinmedia/open-api-framework/issues/64 for it